packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -524,7 +524,7 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 */
 	// tslint:disable-next-line:cognitive-complexity
 	private initializeDataSource(): void {
-		this.dataSource = new MatTableDataSource(this.data);
+		this.dataSource = new MatTableDataSource(this.data || []);
```

Comment:
I'm wondering... isn't it better to set `this.data` to `[]` inside the `ngOnChanges` only instead of repeating `this.data || []` wherever it is used? Then we will only have one single point to look at in case of errors with the `data` input.

Reply:
Indeed, this brings the fix back to 1 line. 🙈